packages/components/psammead-grid/src/index.jsx

```diff
@@ -0,0 +1,219 @@
+    gutterSize: GEL_GUTTER_ABOVE_600PX,
+    marginSize: GEL_MARGIN_ABOVE_400PX,
+  },
+  group5: {
+    min: GEL_GROUP_5_SCREEN_WIDTH_MIN,
+    max: null,
+    gutterSize: GEL_GUTTER_ABOVE_600PX,
+    marginSize: GEL_MARGIN_ABOVE_400PX,
+  },
+};
+
+const GROUP_4_MAX_WIDTH = `45.5rem`;
+// (6.75rem * 6) + 5*16px gutters = 728 = 45.5 rem
+
+const GROUP_5_MAX_WIDTH = `46.4rem`;
+// (2.95rem * 12) + 11*16px gutters = 742.4 = 46.4 rem
+
+const mediaQuery = ({ min, max, styles }) => {
+  if (min && max) {
+    return `
+      @media (min-width: ${min}) and (max-width: ${max}) {
+        ${styles}
+      }
+    `;
+  }
+  if (min) {
+    return `
+      @media (min-width: ${min}) {
+        ${styles}
+      }
+    `;
+  }
+  if (max) {
+    return `
+      @media (max-width: ${max}) {
+        ${styles}
+      }
+    `;
+  }
+  return '';
+};
+
+const gelMaxWidths = css`
+  margin: 0 auto;
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
+    max-width: ${GROUP_4_MAX_WIDTH};
+  }
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    max-width: ${GROUP_5_MAX_WIDTH};
+  }
+`;
+
+const group4MaxWidth = css`
+  margin: 0 auto;
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    max-width: ${GROUP_4_MAX_WIDTH};
+  }
+`;
+
+const gridMediaQueries = ({
+  columns,
+  startOffset,
+  enableGelGutters,
+  enableGelMargins,
+}) => {
+  const selectedGroups = Object.keys(columns);
+  return selectedGroups.map(group =>
+    mediaQuery({
+      min: groups[group].min,
+      max: groups[group].max,
+      styles: `
+        width: initial;
+        grid-template-columns: repeat(${columns[group]}, 1fr);
+        grid-column-end: span ${columns[group]};
+      ${enableGelGutters ? `grid-column-gap: ${groups[group].gutterSize};` : ``}
+      ${enableGelMargins ? `padding: 0 ${groups[group].marginSize};` : ``}
+      ${
+        startOffset && startOffset[group]
+          ? `grid-column-start: ${startOffset[group]};`
+          : ``
+      }`,
+    }),
+  );
+};
+
+/*
+ * 1 We vertically align to the top so that sibling
+ *   grid items that are placed side-by-side within a row
+ *   have their text and images aligned
+ */
+const gridFallbacks = css`
+  ${({ item, columns, parentColumns }) => {
+    if (!item || !parentColumns) {
+      return `position: relative;`;
+    }
+    const selectedGroups = Object.keys(columns);
+    return selectedGroups.map(
+      group =>
+        `
+      ${mediaQuery({
+        min: groups[group].min,
+        max: groups[group].max,
+        styles: `
+          display: inline-block;
+          width: ${(100 * columns[group]) / parentColumns[group]}%;
+          vertical-align: top; `,
+        /* [1] */
+      })}
+    `,
+    );
+  }}
+`;
+
+const GridComponent = styled.div`
+  ${({ enableGelMaxWidths }) => enableGelMaxWidths && gelMaxWidths}
+  ${({ enableGroupFourMaxWidth }) => enableGroupFourMaxWidth && group4MaxWidth}
```

Comment:
@dr3 I'm struggling to understand this. Can you provide a quick code example?

Reply:
I believe that he means to do something like:
```
const Wrapper = styled.div`
   ${({ enableGroupFourMaxWidth }) => enableGroupFourMaxWidth && group4MaxWidth} 
`;

<Wrapper enableGroupFourMaxWidth>
   <Grid columns={{ group3: 6, group4: 8 }}>
        <Grid item columns={{ group3: 6, group4: 6 }}>
           ...
       </Grid>
   </Grid>
</Wrapper>
```

Reply:
Ok thanks. So a separate exported component to constrain and center the grid. What's the benefit of having this as a separate component? Also, if we do that for `enableGroupFourMaxWidth` then why wouldn't we apply the same thinking to `enableGelMaxWidths`?

Reply:
I was more suggesting something like
```
const ArticlePageGridStyle = styled(Grid)`
  margin: 0 auto;
  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
    max-width: ${GROUP_4_MAX_WIDTH};
  }
  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
    max-width: ${GROUP_5_MAX_WIDTH};
  }
`;

export const ArticlePageGrid = ({ children, ...props }) => (
  <ArticlePageGridStyle
    columns={{ group5: 20 }}
    enableGelGutters
    enableGelMargins
    {...props}
  >
    {children}
  </ArticlePageGridStyle>
);
```

but likely not in this PR, we need to decide where to put them

Reply:
I don't really know if there would be a benefit on that, I guess it is more for clarity and cleanliness. If in the future we want to have another page type width, then we would have to update the `GridComponent` to allow to pass a new constraint, rather than create another export and use that wrapper.

I agree that if we change it, it should be applied to both `enableGroupFourMaxWidth` and `enableGelMaxWidths`

Reply:
@jroebu14 @DenisHdz I was more suggesting something like
```
const ArticlePageGridStyle = styled(Grid)`
  margin: 0 auto;
  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
    max-width: ${GROUP_4_MAX_WIDTH};
  }
  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
    max-width: ${GROUP_5_MAX_WIDTH};
  }
`;

export const ArticlePageGrid = ({ children, ...props }) => (
  <ArticlePageGridStyle
    columns={{ group5: 20 }}
    enableGelGutters
    enableGelMargins
    {...props}
  >
    {children}
  </ArticlePageGridStyle>
);
```

but likely not in this PR, we need to decide where to put them

Reply:
@dr3 ok cool. I can create a separate issue to cover this work if that's ok with you

Reply:
@dr3 does this look ok? https://github.com/bbc/simorgh/issues/4054